Specs/Machinetools.md

```diff
@@ -12,36 +12,27 @@ For participants that are familiar with the information model, we have included
 
-### NodeSets for the showcase
+### NodeSets
 
-[Released UaModeler Project](https://github.com/umati/MachineTool-Plugfest/tree/master/UaModelerProject)
-[Released NodeSet](https://github.com/OPCFoundation/UA-Nodeset/tree/v1.04/MachineTool)
+[Normative NodeSet hosted by the OPC Foundation](https://github.com/OPCFoundation/UA-Nodeset/tree/v1.04/MachineTool)
 
-[Protoyping NodeSet](https://github.com/umati/MachineTool-Plugfest/tree/master/UaModelerProtoyping)
 
-**Remark:** You have to be umati partner and added to the access list. (Contact: [info@umati.org](mailto:info@umati.org))
+### For umati Machine Tool partners
 
-### Prototyping vs Non-Prototyping Machine Tools Nodesets
-
-The OPC UA companion specification for Machine Tools uses the *OrderedListType*, which was introduced in [OPC 10001-13 / Errata 1.04.7](https://reference.opcfoundation.org/v104/Core/docs/Amendment13/).
-This specific nodeset can only be integrated to an OPC UA server, which also supports this amendment. As severals SDKs and toolchains are not yet updated to 1.04.7 and do not allow the exchange of namespace 0, we provide the prototyping nodeset which is compatible to all 1.04 Servers. The prototyping nodeset introduces an additional namespace `http://vdw.de/protoyping/`, which adds the required ObjectTypes of the amendment to the address space.
-
-To determine if your toolchain is compatible with the OPC UA companion specification for Machine Tools specification, check for the *OrderedListType* (NodeId `ns0;i=23518`) and have a look at `Root->Objects->Server->Namespaces->http://opcfoundation.org/UA/->NamespaceVersion` (NodeId: `ns0;i=15959`) whether the version is greater or equal to 1.04.7. If the toolchain is not compatible, stick with the prototyping nodeset for the showcase.
-
-The prototyping nodeset is only supported for the upcoming showcases in 2020 and the **support of the prototyping nodeset may be dropped for further showcases** in 2021 when a reasonable amount of SDKs have been updated.
+[UaModeler Project](https://github.com/umati/MachineTool-Plugfest/tree/master/UaModelerProject) as a quick start ( **Remark:** You have to be umati partner and added to the access list. (Contact: [info@umati.org](mailto:info@umati.org)))
 
 ## General information valid for the umati showcase demonstration
 
-| **DISCLAIMER** | For all applications the OPC UA server is to be configured without predefined NodeIDs for the instances. Clients will need to browse Instances or translate BrowsePaths find the NodeID for accessing the address space and available instances. |
+| **DISCLAIMER** | For all applications the OPC UA server is to be configured without predefined NodeIds for the instances. Clients will need to browse Instances or translate BrowsePaths to find the NodeId for accessing the address space and available instances. |
```

Comment:
"Clients will need to browse Instances or translate BrowsePaths to find the NodeId for accessing the address space and available instances." -> "Clients need to utilize the Services  [Browse](https://reference.opcfoundation.org/Core/docs/Part4/5.8.2/) and [TranslateBrowsePathsToNodeIds](https://reference.opcfoundation.org/Core/docs/Part4/5.8.4/) to access nodes of interest.